x/evidence/legacy/v0_40/genesis.pb.go

```diff
@@ -0,0 +1,332 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: cosmos/evidence/genesis.proto
+
+package v040
```

Comment:
I was finding myself manually copying a bunch of proto impls into `/legacy/v0_40/types.go`, including stuff like:
```golang
func (*GenesisState) Descriptor() ([]byte, []int) {
	return fileDescriptor_948832e7dde7529c, []int{0}
}
```

which is ugly. So I decided to just copy this whole file here.

Reply:
Note: for other modules (bank,slashing), only plain structs are being migrated, so I didn't copy their .pb.go files.